src/lib/Containers/Inquiry.tsx

```diff
@@ -55,6 +61,8 @@ const CancelButton = styled.TouchableOpacity`padding-left: 20;`
 const Content = styled.View`
   margin-left: 20;
   margin-right: 20;
+  alignSelf: ${isPad ? "center" : "stretch"};
+  ${isPad ? "width: 472;" : ""};
```

Comment:
It’s unfortunate that this is laid out at 480px on iPone, otherwise we could have just gone with always setting `center` and `max-width: 472` :/

Reply:
I know :( it's just too wide on the phone. 

When I center and add a max width, it just stays on that width, no longer shrinking to fit the screen. I tried it with a `min-width` also, but to no avail.

Reply:
Do you mean that it would be ok as per the design to use the same max width value for both iPhone and iPad?

Reply:
No, I had hoped I could put in a min and a max, such that it would stretch to fill appropriately on the devices, within those two bounds. However, you can't then also `center` the view. (the entire inquiry screen will be left aligned on iPad) 

Basically, all I meant to say is that it is indeed a bit ugly and unfortunate 😂 I tried a bunch of things, and this was the only solution that gave the desired result.

Reply:
That’s fair 👍